install_config/configuring_pipeline_execution.adoc

```diff
@@ -0,0 +1,50 @@
+[[install-config-configuring-pipeline-execution]]
+= Configuring Pipeline Execution
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== Overview
+
+{product-title} utilizes Jenkins to execute continuous development pipelines.  To
+provide a seamless experience for users, the first time a user creates a build
+configuration using the xref:../architecture/core_concepts/builds_and_image_streams.adoc#pipeline-build[pipeline]
+strategy type, a set of Jenkins related resources will be created within the user's project.
```

Comment:
it shouldn't try to instantiate anything because the jenkins service already exists.